tests/Xamarin.Forms-Performance-Integration/Droid/proguard.cfg

```diff
@@ -0,0 +1,2 @@
+-keep class android.support.v7.widget.** { *; }
+-dontwarn android.support.v7.widget.*
```

Comment:
Looks like we need to preserve more classes here. I have created https://github.com/xamarin/xamarin-android/pull/2403 for that.